src/platforms/linux-openwrt.js

```diff
@@ -152,16 +152,39 @@ function getDhcpStatus() {
  * @param {boolean} enabled - Whether or not to enable the DHCP server
  * @returns {boolean} Boolean indicating success of the command.
  */
-function setDhcpStatus(enabled) {
+function setDhcpServerStatus(enabled) {
   if (!uciSet('dhcp.lan.ignore', enabled ? '0' : '1')) {
     return false;
   }
 
+  if (!uciSet('dhcp.lan.dhcpv6', enabled ? 'server' : 'disabled')) {
+    return false;
+  }
+
+  if (!uciSet('dhcp.lan.ra', enabled ? 'server' : 'disabled')) {
+    return false;
+  }
+
   if (!uciCommit('dhcp')) {
     return false;
   }
 
-  const proc = child_process.spawnSync('/etc/init.d/dnsmasq', ['reload']);
+  let proc = child_process.spawnSync('/etc/init.d/dnsmasq', ['stop']);
```

Comment:
Shouldn't this be start if enabled and stop if not?

Reply:
... yes.

Reply:
Ok, fixed.

Reply:
LGTM